blynklib_mp.py

```diff
@@ -353,8 +355,9 @@ def read_response(self, timeout=0.5):
             rsp_data = self.receive(self.rcv_buffer, self.SOCK_TIMEOUT)
             if rsp_data:
                 self._last_rcv_time = ticks_ms()
-                msg_type, msg_id, h_data, msg_args = self.parse_response(rsp_data, self.rcv_buffer)
-                self.process(msg_type, msg_id, h_data, msg_args)
+                while rsp_data:
+                    msg_type, msg_id, h_data, msg_args, rsp_data = self.parse_response(rsp_data, self.rcv_buffer)
```

Comment:
After getting msg_len we re-define rsp_data
as rsp_data minus msg_len bytes from the begin. Hope you are ok with such approach